create-custom-map-style/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "create-custom-map-style",
+  "description": "Create a custom map style with AWS Location Service",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@aws-amplify/ui-react": "^1.0.5",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "aws-amplify": "^3.3.25",
+    "aws-sdk": "^2.881.0",
+    "maplibre-gl": "^1.14.0",
```

Comment:
Please upgrade this.

Reply:
upgraded this to ^1.15.2 and ran an `npm audit fix` in this latest commit